extensions/amp-video/0.1/test/test-amp-video.js

```diff
@@ -434,7 +434,8 @@ describes.realWin('amp-video', {
     });
   });
 
-  it('should forward certain events from video to the amp element', () => {
+  // TODO(aghassemi, #14704): Unskip this test
+  it.skip('should forward certain events from video to the amp element', () => {
```

Comment:
There are three failing tests. You'll need to skip the other two too. See  https://travis-ci.org/ampproject/amphtml/jobs/368262260#L690-L706

Reply:
That's in a [different PR](https://github.com/ampproject/amphtml/pull/14703).

Reply:
@rsimha the other two are in STAMP and https://github.com/ampproject/amphtml/pull/14703 has handling them.

Reply:
In that case, sorry, please proceed :)